Trading/Mode/remote_trading_signals_trading_mode/remote_trading_signals_trading.py

```diff
@@ -410,7 +410,12 @@ async def _create_orders(self, orders_descriptions, symbol):
                 await self._bundle_order(order_description, to_create_orders, ignored_orders, bundled_with,
                                          fees_currency_side, created_groups, symbol)
         # create orders
+        already_handled_order_ids = self.exchange_manager.exchange_personal_data.orders_manager\
```

Comment:
Should we add a global system to enforce not to add an order id twice in the order manager?